### ISSUE TITLE
Make it possible to have multiple backup repositories for a single ssh user

### ISSUE THREAD

**Issue by eauchat** [YunoHost-Apps/borgserver_ynh#47]
It could be nice if it was possible to receive backups in multiple repositories, in the same server, but without having to use different users.
Something like this:
 - borg A --> borgserver:~/sshUser/repoA
 - borg B --> borgserver:~/sshUser/repoB

At the moment, to have multiple repositories, one must install multiple times borgserver, but the install works only if a new user is chosen.
Maybe the way to do this would be to make it possible to install borgserver without creating a custom user for it, just reusing an existing user. But I believe that would make things quite complicated in case of removing the app.

The issue is quite small for the complexity it seems to bring, so I'm not sure at all it would be worth it, but I post it just to leave some trace of reflections in that direction.

**Comment by jose1711** [comment]
:+1: For now you can still perhaps go with custom SSH commands in `~/.ssh/authorized_keys`:
```
command="borg serve --restrict-to-path /path/to/repoA",restrict ssh-ed25519 XXX repoA_key
command="borg serve --restrict-to-path /path/to/repoB",restrict ssh-ed25519 YYY repoB_key
```

**Comment by eauchat** [comment]
Yes, manually allowing more connections work indeed. I was interested in having the full functionalities of the borgerver_ynh app for each repo, meaning error emails when missing backups...
